apps/workspace-host/src/interface.js

```diff
@@ -628,6 +637,7 @@ async function _pullImage(workspace) {
         }
 
         const toDatabase = false;
+        cache.set(`workspaceProgressInit:${workspace_image}`, progressDetailsInit, 1000 * 60 * 60);
```

Comment:
@mwest1066 you wanted to make the expiration here really long, right?

Reply:
Updated this to expire in 24 hours. Let me know if you want me to adjust here!